src/main/java/dev/Roach/fetchers/DealsFetcher.java

```diff
@@ -16,27 +17,27 @@
 import java.nio.file.Files;
 import java.nio.file.Paths;
 import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
 
-
+@Setter
 public class DealsFetcher {
 
-    private final HttpClient client;
+    private HttpClient client = HttpClient.newBuilder().build();
+    private final ObjectMapper objectMapper = new ObjectMapper();
+    private static final String dealsApiUrl = "https://www.cheapshark.com/api/1.0/deals?";
```

Comment:
upper case, same below (final variables can be lower case, but static final should be upper case)